daemon/volumes.go

```diff
@@ -138,6 +139,14 @@ func (daemon *Daemon) registerMountPoints(container *container.Container, hostCo
 		}
 
 		if label.RelabelNeeded(bind.Mode) {
+			if _, err := os.Stat(bind.Source); err != nil {
```

Comment:
You can skip the stat and just mkdirall on the source.  It will not error if it already exists right?